README.md

```diff
@@ -22,7 +22,11 @@ Or install it yourself as:
 
 ```ruby
 
-summary_document = TelephoneAppointments::SummaryDocumentActivity.new(appointment_id: id, owner_uid: uid)
+summary_document = TelephoneAppointments::SummaryDocumentActivity.new(
+  appointment_id: id,
+  owner_uid: uid, 
+  message: message # 'postal' or 'digital'
```

Comment:
@benlovell I've updated the code and commit message based on my thinking. Let me know if it makes sense or not.